packages/onboarding/src/flow-progress/use-flow-progress.ts

```diff
@@ -50,6 +50,17 @@ const flows: Record< string, { [ step: string ]: number } > = {
 		processing: 6,
 		launchpad: 7,
 	},
+	[ ECOMMERCE_FLOW ]: {
+		intro: 0,
+		storeProfiler: 1,
+		storeAddress: 2,
+		domains: 3,
+		designCarousel: 4,
+		siteCreationStep: 5,
+		processing: 5,
+		waitForAtomic: 5,
+		checkPlan: 5,
+	},
```

Comment:
I believe these steps are out of order, which results in the progress bar jumping around unexpectedly.

When I work through the flow, I see the following order:
1. intro
2. storeProfiler
3. designCarousel
4. domains
5. processing (site creation)
6. checkout (not in our list)
7. checkPlan
8. processing (waitForAtomic)
9. storeAddress
```suggestion
	[ ECOMMERCE_FLOW ]: {
		intro: 0,
		storeProfiler: 1,
		storeAddress: 7,
		domains: 4,
		designCarousel: 3,
		siteCreationStep: 5,
		processing: 5,
		waitForAtomic: 6,
		checkPlan: 5,
	},
```